docs/GettingStartedDocs/install_oe_sdk-Windows.md

```diff
@@ -0,0 +1,75 @@
+# Install the Open Enclave SDK NuGet Package
+
+## Platform requirements
+
+- A system with support for SGX1 with Flexible Launch Control (FLC).
+Note: To check if your system has support for SGX1 with or without FLC, please look [here](./SGXSupportLevel.md).
+
+- Windows Server 2016
```

Comment:
Could Windows 10 have FLC support?

Reply:
It should but is not a currently tested scenario. Will say we support it after we have tests.